benchmark/common_utils.hpp

```diff
@@ -62,10 +194,10 @@ const std::array<Transposition, 3> possible_transpositions(
  * @fn to_transpose_enum
  * @brief Translates from a transposition string to an enum.
  */
-static inline Transposition to_transpose_enum(const char* t) {
-  if (t[0] == 't') {
+static inline Transposition to_transpose_enum(std::string t) {
+  if (t == "t") {
```

Comment:
Enum values can be upper case as well.

Reply:
What do you mean? This string is initially read from the CSV file (or from defaults) and is supposed to be in lower case.

Reply:
I prefer to fix a convention and always use the same case, because I don't want a file to look like:
```
n,n,64,64,64
n,N,64,128,64
T,n,64,256,64
...
```

Reply:
Whether or not you generate a csv file like that, unless it's an entirely closed system, a user may write or generate a file with all upper case. Somewhere down the line someone is going to use "T" and not realise it means "n" instead.

Reply:
It doesn't do that anymore. Everything that is not `n`, `t` or `c` now generates the error: `Unrecognized transpose type: x`

Reply:
You're right that using `n` as default is not a good idea, I fixed that